class-tgm-plugin-activation.php

```diff
@@ -1337,6 +1362,22 @@ protected function display_settings_errors() {
 		}
 
 		/**
+		 * Display admin notice if plugins have been force activated.
+		 *
+		 * @since 2.6.0
+		 */
+		public function display_forced_activation_notice() {
+			if ( $force_activated_plugins = get_transient( 'tgmpa_force_activated_plugins' ) ) {
+				?>
+				<div class="notice <?php echo esc_attr( $this->get_admin_notice_class() ); ?> is-dismissible">
```

Comment:
The sanitization is already done within the `get_admin_notice_class()` method. `esc_attr()` is correct in this case.